Neos.Neos/Classes/Fusion/NodeUriImplementation.php

```diff
@@ -129,11 +129,11 @@ public function evaluate()
         } else {
             return '';
         }
-        /** @todo implement us
+        /* TODO implement us see https://github.com/neos/neos-development-collection/issues/4524 {@see \Neos\Neos\ViewHelpers\Uri\NodeViewHelper::resolveNodeAddressFromString} for an example implementation
```

Comment:
"implement us"?
Not new, but since you changed the comment, we could fix this :)

Reply:
^^ yes thus the issue https://github.com/neos/neos-development-collection/issues/4524

one thing at time